src/main/java/org/cascadebot/cascadebot/commands/management/ModuleCommand.java

```diff
@@ -52,7 +52,7 @@ public CascadePermission getPermission() {
 
     @Override
     public String description() {
-        return "interact with modules";
+        return "Interact with modules";
```

Comment:
I'd change interact to something more specific

Reply:
how about this new one?

Reply:
👌